Makefile

```diff
@@ -36,6 +36,12 @@ compile:
 	@echo "==> Building the project"
 	@docker run --rm -it -e GOOS=${OS} -e GOARCH=amd64 -v ${PWD}/:/app -w /app golang:1.13-alpine sh -c 'apk add git && go build -o cmd/do-csi-plugin/${NAME} -ldflags "$(LDFLAGS)" ${PKG}'
 
+.PHONY: check-unused
+check-unused:
```

Comment:
Could we have this task depend on `vendor` for some DRY?

Reply:
Very good suggestion, done.